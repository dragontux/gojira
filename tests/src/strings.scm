(display "Hello, world")
(newline)

(display (eq? (string->symbol "asdf") 'asdf))
(newline)

(display (string-contains "some string" "str"))
(newline)

(display (string-contains "some string" "saywut"))
(newline)

(display (string-append "testing " "this"))
(newline)

(display (symbol->string 'testing))
(newline)
