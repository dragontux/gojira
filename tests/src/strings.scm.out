"Hello, world"
#t
5
#f
"testing this"
"testing"
"testing\"this \\"
"\\\\\"\"\\"
"abc\\d"
